vms/platformvm/state/state_test.go

```diff
@@ -2156,8 +2156,8 @@ func TestGetCurrentValidators(t *testing.T) {
 			},
 		},
 		{
-			name: "sovs only in same subnet",
-			sovs: []SubnetOnlyValidator{
+			name: "L1 validators only in same subnet",
```

Comment:
(no action)
```suggestion
			name: "L1 validators only with the same SubnetID",
```
"L1 validators in the same subnet" seems contradictory